### ISSUE TITLE
Implement automatic adding of jsxImportSource pragma definition - eslint-plugin-emotion

### ISSUE THREAD

**Issue by Emiliano-Bucci** [emotion-js/emotion#2070]
**What**:
Implement the possibility - when using the new fragma `jsxImportSource` - to automatically add the new pragma definition to the top of the file page, like already is implemented with the "old" pragma `/* @jsx jsx */`

**Why**:
This way we don't have to manually add it every time.

**Comment by osdiab** [comment]
agreed - relevant to #2467 , to opt into that toolset i'd need to in the status quo use the JSX pragma, but it's super inconvenient for NextJS since it does set the runtime to automatic, so it appears I do need to use `jsxImportSource` instead.
